src/summarycode/summarizer.py

```diff
@@ -112,7 +112,7 @@ def summarize_codebase(codebase, keep_details, **kwargs):
         ('holders', holder_summarizer),
         ('authors', author_summarizer),
         ('programming_language', language_summarizer),
-        ('packages', package_summarizer),
```

Comment:
For instance here, do not rename "packages" to "package_manifests", instead populate "packages" from the new top-level "packages" attribute.